src/frontends/onnx/frontend/src/op/cum_sum.cpp

```diff
@@ -20,7 +20,12 @@ OutputVector cum_sum(const Node& node) {
     Output<ngraph::Node> axis;
 
     if (inputs.size() > 1) {
-        axis = inputs.at(1);  // optional input, 0-D tensor
+        // optional input, 0-D or 1-D tensor
+        const auto& axis_shape = inputs.at(1).get_partial_shape();
+        NGRAPH_CHECK(axis_shape.rank().get_length() == 0 || axis_shape.rank().get_length() == 1,
```

Comment:
There's no axis input check in the core operator intentionally so I'd rather not add it here either https://github.com/openvinotoolkit/openvino/blob/master/src/core/src/op/cum_sum.cpp#L47. @mitruska any thoughts?